.nvmrc

```diff
@@ -0,0 +1 @@
+lts/dubnium
```

Comment:
Not a version number?

Reply:
It is only for development, so, it's ok. `ts/dubnium` is 10.16 right now.